site/class-1-load/youmghar/index.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<meta charset=utf8>
+<title>@youmghar</title>
+<meta content=width=device-width,initial-scale=1 name=viewport>
+<script src="https://d3js.org/d3.v4.min.js"></script>
+
+<h1>Dinners</h1>
+
+
+<script>
+ 
+  d3.csv('./index.csv', function (err, questions) {
+  if (err) throw err;
+  questions.forEach(function (q) {
+
+    var type = document.createElement('type');
+    var details = document.createElement('details');
+    var day = document.createElement('day');
+    var paragraph = document.createElement('p');
```

Comment:
You’re creating *real* HTML elements with `document.createElement`. So a `<p>` and `<details>` are fine, but HTML doesn’t know about `<day>` or `<type>` elements.